core/remote/remotemodelserver.cpp

```diff
@@ -438,6 +478,7 @@ void RemoteModelServer::layoutChanged()
 void RemoteModelServer::layoutChanged(const QList<QPersistentModelIndex> &parents,
                                       QAbstractItemModel::LayoutChangeHint hint)
 {
+    m_pendingDataChanged.clear();
```

Comment:
Hm, indeed it could be some issues with Qt 5.x and parents not empty.
I guess we want to send all (i would like to avoid sending obsolete data), or remove the entries having a parent in parents.
Will check what make best.